pkg/runtime/runtime.go

```diff
@@ -0,0 +1,79 @@
+package runtime
+
+import (
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	// Imported for side-effects only. This import will cause GOMAXPROCS to be set to the number of vCPUs
+	// allocated to the process if the process is running in a Linux environment (including when its running
+	// in a docker / K8s setup).
+	_ "go.uber.org/automaxprocs"
+)
+
+const (
+	gomaxprocsKey = "GOMAXPROCS"
+)
+
+func init() {
+	defer func() {
+		log.Infof("runtime: final GOMAXPROCS value is: %d", runtime.GOMAXPROCS(0))
```

Comment:
nit: probably better as a `debug` level message? But could be useful to include on status page (and/or in the flare)